src/common/badges-dashboard/badge.css

```diff
@@ -1,22 +1,22 @@
 .app-header-btn {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    border: solid 1px var(--color-brand-primary);
-    border-radius: 2rem;
-    grid-gap: 0.4rem;
-    padding: 0.4rem 1.2rem;
-    font-size: var(--fs-sm);
-    text-decoration: none;
-    width: unset;
-    height: unset;
-    transition: all 0.12s ease-in-out;
-  }
+  display: flex;
```

Comment:
Please confirm you have run the ReactPlay formatting.

Reply:
Yup i formatted it